Tests/test_regressions.py

```diff
@@ -1467,4 +1468,43 @@ def get_class_class(cls):
         self.assertEqual(o.get_self_class(), o.get_class())
         self.assertEqual(o.get_class(), o.get_class_class())
 
+    @unittest.skipIf(is_mono, 'https://github.com/IronLanguages/ironpython3/issues/937')
+    def test_ipy3_gh985(self):
+        """https://github.com/IronLanguages/ironpython3/issues/985"""
+        code = """
+import sys
+
+
+class Foo:
+    def __init__(self, name):
+        self.name = name
+
+    def __del__(self):
+        sys.stdout.write(self.name + '\\n')
+
+
+def func():
+    foo = Foo('Foo 1')
+    bar = Foo('Foo 2')
+    del bar
+
+
+func()
+
+
+foo = Foo('Foo 3')
+foo = Foo('Foo 4')
+        """
+
+        f = open('temp.py', 'w')
```

Comment:
```python
test_script_file = os.path.join(self.test_dir, 'ipy3_gh985.py')
f = open(test_script_file, 'w')
```
would keep `test_regressions` "parallel-safe" (assuming that #983 will be merged in some form at some point in time).

Reply:
Done.